src/main/java/hudson/plugins/ec2/util/MinimumNumberOfInstancesTimeRangeConfig.java

```diff
@@ -0,0 +1,90 @@
+package hudson.plugins.ec2.util;
+
+import net.sf.json.JSONObject;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+public class MinimumNumberOfInstancesTimeRangeConfig {
+
```

Comment:
The field names are excessively long due to the class name prefix. Is this prefix required?